### ISSUE TITLE
In reference docs, link to (or embed) full example sketches that use methods

### ISSUE THREAD

**Issue by toolness** [molleindustria/p5.play#65]
This is a bit more of a "wouldn't it be cool if..." sort of enhancement, but I thought that it would be great if some of the more complex methods in the reference docs actually linked to relevant examples, to provide a better context for their use.

For example, it would be cool if the [reference docs for `Sprite.displace`](http://p5play.molleindustria.org/docs/classes/Sprite.html#method-displace) linked to the [collisions example](http://p5play.molleindustria.org/examples/index.html?fileName=collisions.js).

I suppose an even better alternative might be to do what p5's documentation does, and actually provide an environment for running and tinkering with full example sketches in the documentation itself ([here's an example](http://p5js.org/reference/#/p5/keyIsDown)).

**Comment by quinton-ashley** [comment]
This would be nice! I think I will do this.

**Comment by quinton-ashley** [comment]
I tried to use the p5js-widget for this but it seems that it's impossible to use libraries with it and it also doesn't take advantage of instance mode, each sketch runs in a separate iframe. I think you might've made it before p5.js supported instance mode.

My idea for the new reference pages is for them to have a LOT of mini-sketches. I ended up making my own little p5.js sketch player I call "p5-mini" that I'll release separately. Here it is in action (although actually editing the sketches doesn't work yet). This is the first part of the new reference docs for the Sprite class.
https://molleindustria.github.io/p5.play/v3_ref/sprite.html
